src/test/java/com/jcabi/github/RtStarsTest.java

```diff
@@ -60,14 +61,19 @@ public void checkIfRepoStarred() throws Exception {
             new MkAnswer.Simple(HttpURLConnection.HTTP_NO_CONTENT)
         ).next(new MkAnswer.Simple(HttpURLConnection.HTTP_NOT_FOUND))
             .start();
-        final Stars stars = new RtStars(
-            new ApacheRequest(container.home())
+        final Stars truestars = new RtStars(
+            new ApacheRequest(container.home()),
+            RtStarsTest.repo("someuser", "starredrepo")
         );
         MatcherAssert.assertThat(
-            stars.starred("someuser", "starredrepo"), Matchers.is(true)
+            truestars.starred(), Matchers.is(true)
+        );
+        final Stars falsestars = new RtStars(
```

Comment:
Let's call this `unstarred` instead. See http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names.